build.proj

```diff
@@ -78,11 +78,25 @@
   <ItemGroup>
     <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/winequality-white.csv"
       Url="https://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-white.csv"
-      DestinationFile="$(MSBuildThisFileDirectory)test/data/external/winequality-white.csv" />
-    
-    <TestFile Condition="'$(IncludeBenchmarkData)' == 'true'" Include="$(MSBuildThisFileDirectory)/test/data/external/WikiDetoxAnnotated160kRows.tsv"
+      DestinationFile="$(MSBuildThisFileDirectory)test/data/external/winequality-white.csv" />    
+  </ItemGroup>
+  
+  <ItemGroup Condition="'$(IncludeBenchmarkData)' == 'true'" >    
+    <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/WikiDetoxAnnotated160kRows.tsv"
       Url="http://aka.ms/tlc-resources/benchmarks/WikiDetoxAnnotated160kRows.tsv"
       DestinationFile="$(MSBuildThisFileDirectory)test/data/external/WikiDetoxAnnotated160kRows.tsv" />
+
+    <TestFile Include="$(MSBuildThisFileDirectory)/test/data/external/MSLRWeb10KTrain3.6MRows.tsv"
+      Url="http://aka.ms/tlc-resources/benchmarks/MSLRWeb10KTrain3.6MRows.tsv"
```

Comment:
Can you re-pull the MSLR-WEB10K dataset files?

Reply:
yeah I have changed them locally

Reply:
The dataset files are shorter than before (currently have 720k rows for training & 240k rows for both validate/test). Hence the file names are a bit off.